unit-test/background/pixel.es6.js

```diff
@@ -0,0 +1,93 @@
+
+const pixel = require('../../shared/js/background/pixel.es6')
+const url = 'https://improving.duckduckgo.com/t/'
+const getURLTestCases = [
+    {
+        'pixelName': 'ep',
+        'result': url + 'ep'
+    },
+    {
+        'pixelName': '',
+        'result': undefined
+    },
+    {
+        'pixelName': undefined,
+        'result': undefined
+    }
+]
+const concatParamsTestCases = [
+    {
+        'params': [],
+        'result': ''
+    },
```

Comment:
Is this the behaviour we want? I'd have expected no params to just produce `?100000`

Reply:
oh, welp that's exactly what it did before and it looked wrong so I changed it. Sorry, of course we want the random number to still be attached to the URL -__- my bad

Reply:
Basically I forgot the pixelName is added separately, welp